resources/views/requirements.blade.php

```diff
@@ -362,7 +362,7 @@
         </div>
     </div>
 
-    @yield('entry_btn')
+    @include('components.entry_btn',['entryGtm' => {{requirements_entry_02}},'requirementsGtm' => {{requirements_requirements_02}}])
```

Comment:
@yosuke-fujita 
```suggestion
    @include('components.entry_btn',['entryGtm' => 'requirements_entry_02','requirementsGtm' => 'requirements_requirements_02'])
```
こちらも同じく、波括弧は消して`'`で囲って欲しいです！